1. Was getting this error when main was being called like this 'sys.exit(main(sys.argv[1:]))' :

An exception has occurred, use %tb to see the full traceback. SystemExit

So, I had to change it to main(sys.argv[1:])

2. Also, set EXTRA_CREDIT as 1 if you want to add the extra 4 hearst patterns.

3. Changed file name to mmb180005_ie.py as per previous conventions.